src/Compilers/CSharp/Portable/Binder/Semantics/Operators/BinaryOperatorAnalysisResult.cs

```diff
@@ -36,6 +36,7 @@ public bool HasValue
             get { return this.Kind != OperatorAnalysisResultKind.Undefined; }
         }
 
+        bool IMemberResolutionResultWithPriority<MethodSymbol>.IsApplicable => IsValid;
```

Comment:
nit: consider renaming existing `IsValid` property to `IsApplicable` instead. There's not much benefit of having two names for the same thing.

Reply:
I'd rather keep this scoped. If you want to come back with a refactoring of this I would be happy to review, but I don't want to conflate it with this change.